src/helpers/strings-so.ts

```diff
@@ -0,0 +1,541 @@
+    "ai-assistant-suggestion-sleep-7-days": "Sidee ayay ahayd hurdadaydu 7-dii maalmood ee la soo dhaafay?",
+    "ai-assistant-suggestion-fall-asleep-time-2-weeks": "Waqtigee ayaan caadi ahaan seexday 2-dii toddobaad ee la soo dhaafay?",
+    "ai-assistant-suggestion-sleep-quality-change-month": "Sidee ayay u isbedeshay tayada hurdadayda bishii la soo dhaafay?",
+    "ai-assistant-suggestion-last-tetanus-vaccine": "Goorma ayaa ahaa tallaalkaygii ugu dambeeyay ee teetanada?",
+    "ai-assistant-suggestion-last-blood-test-lab-work": "Goorma ayaa ahaa baadhitaankayga dhiigga ama shaqada shaybaadha ee ugu dambeeyay?",
+    "ai-assistant-suggestion-abnormal-lab-results": "Ma leeyahay wax natiijooyinka shaybaadha ah oo aan caadi ahayn?",
+    "ai-assistant-suggestion-last-cbc-test": "Goorma ayaa ahaa tirada dhiiggayga dhammaystiran (CBC) ee ugu dambeeyay?",
+    "ai-assistant-suggestion-glucose-a1c-levels-last-test": "Waa maxay heerarka sonkorowga iyo A1c ee baadhitaankayga ugu dambeeyay?",
+    "ai-assistant-suggestion-graph-cholesterol-trends": "Ma i tusi kartaa jaantus muujinaya sida heerarka kolesteroolkayga u isbedeleen waqti ka dib?",
+    "ai-assistant-suggestion-last-metabolic-panel": "Goorma ayaa la sameeyay baadhitaankayga metabolic-ka ee ugu dambeeyay?",
+    "ai-assistant-suggestion-hemoglobin-levels-trend": "Waa maxay isbedelka heerarka hemoglobin-kayga?",
+    "ai-assistant-suggestion-show-files": "I tus faylashayda.",
+    "ai-assistant-suggestion-save-graph-to-files": "Ku kaydi jaantuska faylashayda",
+    "mindful-minutes": "Daqiiqadaha Feejignaanta",
+    "therapy-minutes": "Daqiiqadaha Daaweynta",
+    "insight-matrix-no-comparison-data": "Ma jiraan noocyada xogta isbarbardhigga ee la habeeyay.",
+    "allergylist-reactions": "Falcelinta",
+    "points-abbreviation": "dhbc",
+    "no-data": "Xog Ma Jirto",
+    "no-data-yet": "Weli xog ma jirto",
+    "bp-low": "Hoose",
+    "bp-normal": "Caadi",
+    "bp-elevated": "Kor u kacay",
+    "bp-stage1": "Heerka 1",
+    "bp-stage2": "Heerka 2",
+    "bp-crisis": "Xaalad degdeg ah",
+    "bp-unknown": "Aan la garanayn",
+    "device-not-enabled": "@@DEVICE@@ looma oggolaan mashruucan.",
+    "download-pdf-report": "Soo dejiso Warbixinta PDF",
+    "connect-to-device": "Ku xidh @@DEVICE@@",
+    "symptoms-and-treatments": "Astaamaha & Daaweynta",
+    "subsequent-evaluation-note": "Qoraalka qiimaynta xigta",
+    "summary": "Khulaasa",
+    "device-activity": "Dhaqdhaqaaqa Qalabka",
+    "daily": "Maalinle",
+    "weekly": "Toddobaadle",
+    "monthly": "Bile",
+    "bonus": "Dheeraad",
+    "syncing-data": "Xogta la isku xidhayo...",
+    "health-connect-phr-sync-title": "La xidhiidh Health Connect",
+    "health-connect-phr-sync-prompt": "Dooro diiwaannada caafimaadka ee laga akhriyo oo loo qoro Health Connect"
+};
+
+export default strings;
+
+
+
+
```

Comment:
This file and a few of the other new language files have some unnecessary blank lines at the end.